girder/api/docs.py

```diff
@@ -112,13 +112,25 @@ def removeRouteDocs(resource, route, method, info, handler):
                 del routes[resource]
 
 
-def addModel(name, model):
+def addModel(resource, name, model):
     """
-    This is called to add a model to the swagger documentation.
+    Add a model to the Swagger documentation.
 
+    :param resource: The type of existing resource to add the model to.
+    :param resource: str
     :param name: The name of the model.
     :type name: str
     :param model: The model to add.
     :type model: dict
```

Comment:
+1 to @cjh1 's solution, if @cpatrick and the other @girder/owners can tolerate this minor breach of API comparability orthodoxy.

We don't even need a separate method to do the multi-resource thingy. The ``resource`` argument can just accept a ``str`` or `tuple of str` and behave appropriately. I'll push that change now.